docs/api/alerting/create_rule.asciidoc

```diff
@@ -18,12 +18,14 @@ WARNING: This API supports <<token-api-authentication>> only.
 [[create-rule-api-path-params]]
 ==== Path parameters
 
-`<id>`::
-  (Optional, string) Specifies a UUID v1 or v4 to use instead of a randomly generated ID.
-
 `space_id`::
   (Optional, string) An identifier for the space. If `space_id` is not provided in the URL, the default space is used.
 
+WARNING: As part of the {kibana-ref-all}/master/sharing-saved-objects.html[Sharing Saved Objects] effort, IDs for rules in a custom space will be regenerated in 8.0.0. Rules created using this API that specify both the `id` and `space_id` path parameters will be re-assigned a randomly generated ID.
```

Comment:
**nit:** I interpreted the second sentence as in my rule would get a randomly generated ID after calling the API (and not after upgrading to 8.0). Not sure if others read it like that?

Reply:
Updated to be clearer about when re-assignment occurs in https://github.com/elastic/kibana/pull/108552/commits/c8cef16137469b3a1f098608b6dd94f3f8c46b49

Reply:
Thanks! LGTM!

Reply:
I wonder if you can pare this down a bit:

If you specify both the `id` and `space_id` path parameter when you use this API to create a rule, the ID will be regenerated when you upgrade to 8.0.0. Learn more.

where Learn more links to the details in the Shared Saved Objects doc.